rskj-core/src/test/java/co/rsk/peg/BridgeTestIntegration.java

```diff
@@ -1042,7 +1031,6 @@ public void getMinimumLockTxValue() throws Exception{
         Bridge bridge = new Bridge(PrecompiledContracts.BRIDGE_ADDR, constants, activationConfig,
                 bridgeSupportFactory);
         Transaction mockedTx = mock(Transaction.class);
-        when(mockedTx.isLocalCallTransaction()).thenReturn(true);
```

Comment:
You accidentally, or not, uncover an issue with this test. This test should fail if you don't make the `isLocalCallTransaction` method to return true. But only if RSKIP_88 is active.
If you add:
```
doReturn(true).when(activationConfig).isActive(eq(RSKIP88), anyLong());
```
at the beginning of this method, alongside your line you removed, this test will fail.
We should have added another test proving this (expecting the test to fail). We can work on this after you make the merge.
cc: @marcos-iov

Reply:
I would say accidentally :)

It was the new version of Mockito that spotted this line as an "unused stub".